tutorial/practicals/LFRic/distributed_memory/2_annexed_dofs/Makefile

```diff
@@ -37,8 +37,8 @@
 include ../../../../common.mk
 
 transform:
-	$(PSYCLONE) -s ./schedule.py -oalg /dev/null -opsy /dev/null ../code/helmholtz_solver_alg_mod.x90
-	psyclone -s ./schedule.py -oalg /dev/null -opsy /dev/null ../code/helmholtz_solver_alg_mod.x90 --config psyclone.cfg
+	$(PSYCLONE) -api lfric -s ./schedule.py -oalg /dev/null -opsy /dev/null ../code/helmholtz_solver_alg_mod.x90
```

Comment:
It is not, but there is a comment that it has to be modified as part of the exercise. I added the comment. I also removed the psyclone call without config file since its already the the previous exercise.